src/test/FuzzerImpl.cpp

```diff
@@ -911,38 +916,150 @@ auto constexpr DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY = 256;
 std::array<
     AccountParameters,
     FuzzUtils::NUMBER_OF_PREGENERATED_ACCOUNTS> constexpr accountParameters{
-    {{DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY,
-      {AssetID(1), AssetID(2), AssetID(3), AssetID(4)}},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY,
-      {AssetID(), AssetID(2), AssetID(3), AssetID(4)}},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY,
-      {AssetID(1), AssetID(), AssetID(3), AssetID(4)}},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY,
-      {AssetID(1), AssetID(2), AssetID(), AssetID(4)}},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY,
-      {AssetID(1), AssetID(2), AssetID(3), AssetID()}},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY,
-      {AssetID(1), AssetID(2), AssetID(3), AssetID(4)}},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY,
-      {AssetID(1), AssetID(2), AssetID(3), AssetID(4)}},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY,
-      {AssetID(1), AssetID(2), AssetID(3), AssetID(4)}},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY,
-      {AssetID(1), AssetID(2), AssetID(3), AssetID(4)}},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY,
-      {AssetID(1), AssetID(2), AssetID(3), AssetID(4)}},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY,
-      {AssetID(1), AssetID(2), AssetID(3), AssetID(4)}},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY,
-      {AssetID(1), AssetID(2), AssetID(3), AssetID(4)}},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY,
-      {AssetID(1), AssetID(2), AssetID(3), AssetID(4)}},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY,
-      {AssetID(1), AssetID(2), AssetID(3), AssetID(4)}},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY,
-      {AssetID(1), AssetID(2), AssetID(3), AssetID(4)}},
-     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY,
-      {AssetID(1), AssetID(2), AssetID(3), AssetID(4)}}}};
+    {{DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY, // account 0
+      {TrustLineParameters(AssetID(1),
+                           DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY),
+       TrustLineParameters(AssetID(2),
+                           DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY),
+       TrustLineParameters(AssetID(3),
+                           DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY),
+       TrustLineParameters(AssetID(4),
+                           DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY)}},
+     {DEFAULT_ASSET_AVAILABLE_FOR_TEST_ACTIVITY, // account 1
+      {TrustLineParameters(AssetID(),
```

Comment:
I only realized this after reviewing everything, but this interface isn't great. It's forcing you to specify parameters for a trust line that doesn't exist.

Reply:
Good point -- I added a default constructor, and also one for `ClaimableBalanceParameters` for the same reason.